packages/ui/src/containers/result-data/experimentation-guide.tsx

```diff
@@ -110,6 +111,14 @@ export const ExperimentationGuide = (props: ResultDataProps) => {
   ) : (
     <Box p={2}>Please run optimizer</Box>
   )
+
+  const debouncedUpdate = _.debounce(suggestionCount => {
```

Comment:
debouncedUpdate should probably be wrapped in useCallback to prevent the function to be recreated on every render

Reply:
Also, lodash debounce can be installed with ```npm install lodash.debounce``` to reduce bundle size